core/src/main/java/bisq/core/app/BisqSetup.java

```diff
@@ -559,7 +576,7 @@ else if (displayTorNetworkSettingsHandler != null)
 
         // We only init wallet service here if not using Tor for bitcoinj.
         // When using Tor, wallet init must be deferred until Tor is ready.
-        if (!preferences.getUseTorForBitcoinJ() || localBitcoinNode.isDetected()) {
```

Comment:
Removed redundant `LocalBitcoinNode` call, because `Preferences.getUseTorForBitcoinJ` makes the same call and it has the same effect.

Reply:
You're right, I missed that. My concern here is that this line decides whether to "initiate wallet" at this time, which implies that Tor will not be used. I think ideally `Preferences.getUseTorForBitcoinJ()` would be a full authority on whether or not we're using Tor, but here it's only one half of the condition.